docs/ops.html

```diff
@@ -3482,8 +3482,24 @@ <h3 class="anchor-heading"><a id="zk" class="anchor-link"></a><a href="#zk">6.9
 
   <h4 class="anchor-heading"><a id="zkversion" class="anchor-link"></a><a href="#zkversion">Stable version</a></h4>
   The current stable branch is 3.5. Kafka is regularly updated to include the latest release in the 3.5 series.
+  
+  <h4 class="anchor-heading"><a href="#zk_depr" class="anchor-link">ZooKeeper Deprecation</a></h4>
 
-  <h4 class="anchor-heading"><a id="zkops" class="anchor-link"></a><a href="#zkops">Operationalizing ZooKeeper</a></h4>
+  	<p>With the release of Apache Kafka 3.5, Zookeeper is now marked deprecated. Removal of ZooKeeper is planned in the next major release of Apache Kafka, which is scheduled to happen no sooner than version 4.0. During the deprecation phase, ZooKeeper is still supported for metadata management of Kafka clusters, but it is not recommended for new deployments. There is a small subset of features that remain to be implemented in KRaft, see <a href="#kraft_missing" class="anchor-link">current missing features</a> for more information.</p>
```

Comment:
"in the next major release of Apache Kafka, which is scheduled to happen no sooner than version 4.0" should probably be "in the next major release of Apache Kafka (version 4.0), which is scheduled to happen no sooner than April 2024".